src/components/Calculator.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import Button from './Button';
+
+const Calculator = () => (
+  <div className="calculator">
+    <input type="text" id="result" readOnly value={0} />
+    <div className="buttons">
+      <Button className="operation">+/-</Button>
+      <Button className="operation">+/-</Button>
+      <Button className="operation">%</Button>
+      <Button className="operation">÷</Button>
+
+      <Button>7</Button>
+      <Button>8</Button>
+      <Button>9</Button>
+      <Button className="operation">X</Button>
+      <Button>4</Button>
+      <Button>5</Button>
+      <Button>6</Button>
+      <Button className="operation">-</Button>
+
+      <Button>1</Button>
+      <Button>2</Button>
+      <Button>3</Button>
+      <Button className="operation">+</Button>
+      <Button>0</Button>
+      <Button>.</Button>
+      <Button className="operation equal">=</Button>
+    </div>
+  </div>
+);
```

Comment:
- [ ] Kindly follow the given design as much as you can because it's part of the project requirements. I have attached two images for your reference, one is your design and the other one is the original.

## Yours
![math](https://github.com/ybadr99/Math_Magicians/assets/53124548/8d618d01-814b-4d28-afad-17ecb278338e)

## Original
![og](https://github.com/ybadr99/Math_Magicians/assets/53124548/d55b1464-579b-4270-9f2c-9127510b0cb8)